packages/client/src/api/forumApi/forumApi.ts

```diff
@@ -0,0 +1,49 @@
+import { baseAxiosInstance, forumAxiosInstance } from '@api/baseAxiosInstance';
+
+const getAllForums = () => {
+  return forumAxiosInstance.get('/forums');
+};
+
+const getAllTopics = (forumId: string) => {
+  return forumAxiosInstance.get(`/forumTopics?forumId=${forumId}`);
+};
+
+interface ICreateNewTopicData {
+  forumId?: string;
+  name: string;
+  yaId: string;
+  authorName: string;
+}
+
+const createNewTopic = (responseData: ICreateNewTopicData) => {
+  return forumAxiosInstance.post('/forumTopics', responseData);
+};
+
+const getTopicComments = (id: string) => {
```

Comment:
Ну и тут же аналогично можно сделать :)
```suggestion
const getTopicComments = (topicId: string) => {
```

P.S. не критично. Up to u